packages/block-library/src/pullquote/edit.js

```diff
@@ -129,9 +65,9 @@ function PullQuoteEdit( {
 							// translators: placeholder text used for the quote
 							__( 'Add quote' )
 						}
-						textAlign="center"
+						textAlign="right"
```

Comment:
How come we're changing this to `right`? 🤔

Reply:
Bad commit. I was testing something. Thanks for spotting. 👍